posthog/management/commands/setup_dev.py

```diff
@@ -1,7 +1,7 @@
 from django.core.management.base import BaseCommand
 from django.db import transaction
 
-from posthog.demo import ORGANIZATION_NAME, TEAM_NAME, create_demo_data
+from posthog.demo.legacy import ORGANIZATION_NAME, TEAM_NAME, create_demo_data
```

Comment:
Should we output that setup_dev is deprecated and people should use generate_demo_data?

Reply:
(GitHub wouldn't let me comment on a more sensible line in the file 🤷 )

Reply:
👍 Good idea